web/elm/src/Build/Header/Header.elm

```diff
@@ -394,12 +395,23 @@ update msg ( model, effects ) =
     case msg of
         ScrollBuilds event ->
             let
+                scrollFactor =
+                    case event.deltaMode of
+                        DeltaModePixel ->
```

Comment:
Just a side comment, not super important, how'd you get the 20 / 800 for line / page?

Reply:
Totally fair question! Tried to give some justification for it in the commit message I added that in.

tl;dr 20 for Line is based on the font-size of the scroll bar. The 800 for Page is totally arbitrary - I took it from some Facebook library. I haven’t been able to produce a DeltaModePage event, so not sure if it feels natural, but it’s better than nothing!